nestkernel/event.h

```diff
@@ -582,18 +587,22 @@ class DataLoggingRequest : public Event
 inline DataLoggingRequest::DataLoggingRequest()
   : Event()
   , recording_interval_( Time::neg_inf() )
+  , recording_offset_( Time::ms( 0. ) )
```

Comment:
This doesn't work.. initializing recording_offset_ with ( 0 ) leads to a compile error.